src/codemagic/apple/resources/resource.py

```diff
@@ -218,7 +224,9 @@ def to_iso_8601(cls, dt: datetime) -> str:
     def to_iso_8601(cls, dt: Optional[datetime]):
         if dt is None:
             return None
-        return dt.strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + '+0000'
+        if dt.tzinfo == timezone.utc:
+            return dt.strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + '+0000'
+        return dt.isoformat()
```

Comment:
apparently new format, build api returns `'2021-01-28T06:01:32-08:00'`, so change the output so that Build -> dict works fine

Reply:
added an annotation [here](https://github.com/codemagic-ci-cd/cli-tools/pull/67/commits/71f2ca10806aa46be85d0feca84182780939f3d7#diff-b736b487114b15d1320d4be383e9cb605e79ac0e683d6949d012fc6215dcf44eR227)